mollie/api/resources/base.py

```diff
@@ -36,37 +49,34 @@ def perform_api_call(self, http_method, path, data=None, params=None):
                 )
         return result
 
-    def _validate_resource_id(self, resource_id: str, name: str = "", message: str = ""):
-        resource_name = name or "Identifier"
-        message = (
-            message or f"Invalid {resource_name} '{resource_id}', it should start with '{self.RESOURCE_ID_PREFIX}'."
-        )
+    def validate_resource_id(self, resource_id: str, name: str = "Identifier", message: str = ""):
```

Comment:
Why did you remove the underscore from the method name? It still seems to be a private method even though it is inherited by the other resources.

Reply:
I thought this could be useful for developers implementing the library to validate user input and such.